dcos_installer/test_backend.py

```diff
@@ -297,54 +274,50 @@ def test_do_aws_configure(tmpdir, monkeypatch):
         assert backend.do_aws_cf_configure() == 0
 
 
-valid_storage_config = """---
+@pytest.fixture
+def valid_storage_config(release_config_aws):
+    """ Uses the settings from dcos-release.config.yaml ['testing'] to create a
+    new upload and then deletes it when the test is over
+    """
+    s3_bucket_name = release_config_aws['bucket']
+    bucket_path = str(uuid.uuid4())
```

Comment:
`uuid.uuid4().encode('ascii')` - please.

Reply:
I'm confused... UUID object does not natively have an 'encode' method, so are you asking me to do `str(uuid.uuid4()).encode('ascii')`? That serves no purpose because that will have to be decoded as a python string when tit is used later on... This seems like a very unnecessary request.

Reply:
My mistake. I mistakenly assumed `uuid4()` returned a byte-string.